src/common/legacy_config_opts.h

```diff
@@ -263,6 +263,7 @@ OPTION(mon_reweight_max_change, OPT_DOUBLE)
 OPTION(mon_health_to_clog, OPT_BOOL)
 OPTION(mon_health_to_clog_interval, OPT_INT)
 OPTION(mon_health_to_clog_tick_interval, OPT_DOUBLE)
+OPTION(mon_health_detail_to_clog, OPT_BOOL)
```

Comment:
nit, i'd suggest avoid adding options used in non-critical paths to `legacy_config_opt.h`.